master/internal/experiment/bulk_action.go

```diff
@@ -44,6 +45,11 @@ type deleteExperimentOKResult struct {
 // ExperimentsAddr is the address to direct experiment actions.
 var ExperimentsAddr = actor.Addr("experiments")
 
+func expNotFoundErr(id int32) error {
```

Comment:
I think you mentioned you wanted to put these off in the api package? that sounds great. 
may I suggest `notFoundErr(name string, id string, rbacControlled bool)`  which can interally handle adding rbac